### PR TITLE
LP-2698 Revoke webinar registration delete permission from ADG admin

### PR DESCRIPTION
**Pull request by aliassad**
Revoke webinar registration delete permission from ADG admin

### CODE REVIEW

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/1876/workflows/workflows_id/jobs/1876/artifacts)**. Coverage !!! :scream:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/webinars/admin.py (50.0%): Missing lines 230
-------------
Total:   2 lines
Missing: 1 line
Coverage: 50%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/1877/workflows/workflows_id/jobs/1877/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/webinars/admin.py (100%)
openedx/adg/lms/webinars/tests/test_admin.py (100%)
-------------
Total:   4 lines
Missing: 0 lines
Coverage: 100%
-------------
```

**Review by MunamMubashir-arbisoft — APPROVED** [review]
LGTM!
Added a pylint error related suggestion.  Please have a look at it as well!

File: openedx/adg/lms/webinars/tests/test_admin.py [hunk 1]
```diff
@@ -74,6 +74,13 @@ def test_cancelled_webinar_admin_delete_permission():
     assert not CancelledWebinarAdmin.has_delete_permission('self', Mock())
 
 
+def test_webinar_registration_admin_delete_permission():
+    """
+    Test if WebinarRegistrationAdmin does not have delete permission
```
Comment (harris-irfan) [inline openedx/adg/lms/webinars/tests/test_admin.py:79]: *Test that ADG admins do not have permission to delete webinar registrations

**Review by harris-irfan — CHANGES_REQUESTED** [review]

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/1881/workflows/workflows_id/jobs/1881/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/webinars/admin.py (100%)
openedx/adg/lms/webinars/tests/test_admin.py (100%)
-------------
Total:   4 lines
Missing: 0 lines
Coverage: 100%
-------------
```

**Review by javeriaejaz-arbisoft — APPROVED** [review]
LGTM :+1:

**Comment by MunamMubashir-arbisoft** [comment]
> **pylint:** ❌
> **pycodestyle:** ✔️
> **iSort:** ✔️
> 
> **Please fix following pylint issues:**
> 
> ```
>  ************* Module edx-platform.openedx.adg.lms.webinars.tests.test_admin
> openedx/adg/lms/webinars/tests/test_admin.py:274:0: E0102: function already defined line 246 (function-redefined)
> 
> ------------------------------------------------------------------
> Your code has been rated at 9.78/10 (previous run: 9.78/10, +0.00) 
> ```

Can we kindly so something to avoid this pylint error?
If we can do nothing else, can we please suppress it via pylint disable? @aliassad

**Pull request closed without merging**